include/a3/editor_f/Data/Scripts/dikCodes.h

```diff
@@ -0,0 +1,181 @@
+#ifndef DIK_ESCAPE
```

Comment:
This is the wrong file.
https://github.com/acemod/ACE3/blob/master/include/a3/ui_f/hpp/defineDIKCodes.inc
That's the correct one. The ui_f one.